pom.xml

```diff
@@ -998,6 +1015,33 @@
                 <artifactId>jedis</artifactId>
                 <version>${version.lib.jedis}</version>
             </dependency>
+            <dependency>
+                <groupId>org.jboss.narayana.jta</groupId>
+                <artifactId>cdi</artifactId>
+                <version>${version.lib.narayana}</version>
+                <type>jar</type>
```

Comment:
Can you remove type=jar, this follows the same pattern as the other dependencies.

Reply:
Done.